Cargo.toml

```diff
@@ -42,9 +42,9 @@ default-members = [
     "wgpu",
 ]
 
-[workspace.lints.clippy]
-manual_c_str_literals = "allow"
-ref_as_ptr = "warn"
+[workspace.lints.rust]
```

Comment:
This is incorrect, these are both clippy lints.

Reply:
When I tried moving these two to `clippy.toml`, Clippy complained about these. This is why I thought these should be rust lints.

Reply:
The clippy config file seems to only contain things that can be changed about lints - it feels kinda weird and half baked https://doc.rust-lang.org/clippy/lint_configuration.html